packages/types/src/definitions/runtime/MessageControl.type.ts

```diff
@@ -2,3 +2,10 @@ export type Message = {
   text: string;
   visible: boolean;
 };
+
+export type TimerMessage = {
+  text: string;
+  visible: boolean;
+  timerBlink: boolean;
+  timerBlackout: boolean;
+}
```

Comment:
we should tie these two definitions together so that TimerMessage is an extension of Message, like

```
export type TimerMessage = Message & {
  timerBlink: boolean;
  timerBlackout: boolean;
}
```